build/bootstrap.proj

```diff
@@ -0,0 +1,31 @@
+<Project Sdk="Microsoft.Build.NoTargets/1.0.80">
+    <PropertyGroup>
+        <RestorePackagesPath>$(MSBuildThisFileDirectory)../packages/</RestorePackagesPath>
+        <BaseIntermediateOutputPath>$(RestorePackagesPath)obj/</BaseIntermediateOutputPath>
+        <IntermediateOutputPath>$(BaseIntermediateOutputPath)$(Configuration)/</IntermediateOutputPath>
+        <TargetFramework>netstandard2.0</TargetFramework>
+        <DisableImplicitNuGetFallbackFolder>true</DisableImplicitNuGetFallbackFolder>
+    </PropertyGroup>
+
+    <ItemGroup>
+        <PackageDownload Include="ILMerge" version="[3.0.21]" />
+        <PackageDownload Include="Lucene.Net" version="[3.0.3]" />
+        <PackageDownload Include="MicroBuild.Core" version="[0.2.0]" />
+        <PackageDownload Include="Microsoft.Build" version="[15.1.262-preview5]" />
+        <PackageDownload Include="Microsoft.DotNet.Build.Tasks.Feed" version="[2.2.0-beta.19178.1]" />  <!-- for publishing to the .NET Core build asset registry (BAR) -->
+        <PackageDownload Include="Microsoft.DotNet.Maestro.Tasks" version="[1.1.0-beta.19178.1]" />  <!-- for publishing to the .NET Core build asset registry (BAR) -->
+        <PackageDownload Include="Microsoft.VisualStudio.Composition" version="[15.8.98]" />
+        <PackageDownload Include="Microsoft.VisualStudio.ProjectSystem" version="[16.0.201-pre-g7d366164d0]" />
+        <PackageDownload Include="Microsoft.VSSDK.BuildTools" version="[15.9.3032]" />
+        <PackageDownload Include="Microsoft.Web.Xdt" version="[2.1.2]" />
```

Comment:
Now that this is a not packages.config anymore we can consider a having the versions in common.props or common.project.props or something that. 
Specifically this applies for packages that are both in the bootstrap and as a PackageReference. 

So Web.Xdt, Newtonsoft.Json are definitely some of those. 

Then again, you can also do all of them, then we'll never need to change any of the includes such as

 `<ILMergeExePath>$(SolutionPackagesFolder)ilmerge\3.0.21\tools\net452\ILMerge.exe</ILMergeExePath>`

Reply:
Reviewing further down, we have the exact path of the assemblies in the packages, so maybe it doesn't matter as much :)

Reply:
Some bootstrap packages need some more adjustments when they are referenced in other projects. For example:

https://github.com/NuGet/NuGet.Client/blob/840b09a4cecda6ae8fd1c07967874fe8061ece63/src/NuGet.Clients/NuGet.VisualStudio.Client/NuGet.VisualStudio.Client.csproj#L71

With this PR, the path would change to:

```
 <Content Include="$(SolutionPackagesFolder)Lucene.Net\3.0.3\lib\NET40\Lucene.Net.dll"> 
```

Reply:
yeah, that's what I meant in the 2nd comment. 

I don't feel too strongly either way, so make a judgment call